test/extended/prometheus/prometheus.go

```diff
@@ -165,11 +165,8 @@ var _ = g.Describe("[Feature:Prometheus][Conformance] Prometheus", func() {
 			execPod := exutil.CreateCentosExecPodOrFail(oc.AdminKubeClient(), ns, "execpod", nil)
 			defer func() { oc.AdminKubeClient().CoreV1().Pods(ns).Delete(execPod.Name, metav1.NewDeleteOptions(1)) }()
 
-			tests := map[string][]metricTest{
-				// should have a constantly firing watchdog alert
-				`ALERTS{alertstate="firing",alertname="Watchdog"}`: {metricTest{greaterThanEqual: true, value: 1}},
-				// should be only one watchdog alert (this is a workaround as metricTest doesn't offer equality operator)
-				`ALERTS{alertstate="firing",alertname="Watchdog",severity="none"}`: {metricTest{greaterThanEqual: false, value: 2}},
+			tests := map[string]bool{
+				`ALERTS{alertstate="firing",alertname="Watchdog"} == 1`: true,
```

Comment:
Why did you remove the `severity="none"` here?

Reply:
I could re-add it, but it won't change anything.